src/Components/drills/FrisbeeDrillIllustration.js

```diff
@@ -54,45 +52,75 @@ const FrisbeeDrillIllustration = (props) => {
     );
   };
 
-  const displayVimeo = ({ illustrationSource, sounds }) => {
+  const displayVimeo = ({ vimeoId, sounds, instruction }) => {
     return (
       <View style={styles.contentWrapper}>
         <View style={[{ height: 250 }, styles.videoAlone]}>
-          <VimeoVideo vimeoId={illustrationSource} sounds={sounds} />
+          <VimeoVideo vimeoId={vimeoId} sounds={sounds} />
         </View>
+        <Text style={styles.instruction}>{instruction}</Text>
       </View>
     );
   };
 
-  const displayAnimation = ({ illustrationSource, instruction }) => {
+  const displayAnimation = ({ animation, instruction }) => {
     return (
       <>
         <View style={styles.contentWrapper}>
-          <Animation widthRatio={1} heightRatio={1 / 2} animation={new Drill(illustrationSource)} />
+          <Animation widthRatio={1} heightRatio={1 / 2} animation={new Drill(animation)} />
         </View>
         <Text style={styles.instruction}>{instruction}</Text>
```

Comment:
Maybe move the instruction out of these displayX functions, to be able to put the toggle between the animation/video and the instruction